.evergreen.yml

```diff
@@ -230,8 +230,8 @@ github_checks_aliases:
 # Triggered on git tag
 git_tag_aliases:
   - git_tag: "^(\\d+\\.)?(\\d+\\.)?(\\d+)$"
-    variant_tags: [ "release" ]
-    task_tags: [ "image_release", "image_preflight", "openshift_bundles", "code_snippets" ]
+    variant_tags: [ "release", "e2e_smoke_release_test_suite" ]
```

Comment:
Is my understanding correct that the release can be started without depending on successful e2e tests?

Reply:
yes, `e2e_smoke_release_test_suite` variant is only depending on `init_test_run` and nothing will block other release tasks to run